src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              apple
```

Comment:
Don't use spaces in the href property of the anchor tag. If it's meant to be a placeholder, consider using '#apple' to represent the link target.